reaper.go

```diff
@@ -249,7 +249,7 @@ func newReaper(ctx context.Context, sessionID string, provider ReaperProvider) (
 		ExposedPorts: []string{string(listeningPort)},
 		Labels:       core.DefaultLabels(sessionID),
 		Privileged:   tcConfig.RyukPrivileged,
-		WaitingFor:   wait.ForListeningPort(listeningPort),
+		WaitingFor:   wait.ForLog("Started"),
```

Comment:
bug: If the listening for check doesn't work then the reaper wont be accessible so this is not a good fix.

Reply:
It could work by adding TESTCONTAINERS_RYUK_ENDPOINT_OVERRIDE_BY_NAME=true, allowing us to contact the container directly through its name. 
I'm not sure it's the most elegant solution, so I'm open to any alternatives you think would be better.